labels.mss

```diff
@@ -1,11 +1,12 @@
 #country[prio<=3][zoom=3],
 #country[prio<=4][zoom=4],
 #country[zoom>=5][zoom<=10] {
-  text-name: '[name]';
+  text-transform: uppercase;
```

Comment:
My previous remark about uppercase not being compatible with other alphabet is still valid ;)

Reply:
Ok let's abandon it for the moment until I receive a complaint